src/Component/CurrentWeather.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import clear from '../img/weather-icons/clear.svg';
+import cloudy from '../img/weather-icons/cloudy.svg';
+import drizzle from '../img/weather-icons/drizzle.svg';
+import fog from '../img/weather-icons/fog.svg';
+import mostlycloudy from '../img/weather-icons/mostlycloudy.svg';
+import partlycloudy from '../img/weather-icons/partlycloudy.svg';
+import rain from '../img/weather-icons/rain.svg';
+import snow from '../img/weather-icons/snow.svg';
+import storm from '../img/weather-icons/storm.svg';
+import unknown from '../img/weather-icons/unknown.svg';
+
+class CurrentWeather extends Component {
+  render() {
+    console.log(this.props.data);
```

Comment:
Next challenge: Use your data inside this component 😃 

Can you see which part of the data you will need for the current weather? Do you need all of it?